platform/lib/samples/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
We should update the comments in all files to match the other amp files.

Reply:
done